### ISSUE TITLE
Compile bug: Build hangs in vulkan-shaders-gen

### ISSUE THREAD

**Issue by yurivict** [ggerganov/llama.cpp#10717]
### Git commit

4285

### Operating systems

BSD

### GGML backends

Vulkan

### Problem description & steps to reproduce

vulkan-shaders-gen hangs during build in the ```uwait ``` state.

```top``` shows this process as:
```
  PID USERNAME    THR PRI NICE   SIZE    RES STATE    C   TIME    WCPU COMMAND
70165 yuri          1  68    0    94M  8212K uwait    1   0:01   0.00% vulkan-shaders-gen
```

```pstree``` shows this process as:
```
 |           | |         \--= 70165 yuri /usr/ports/misc/llama-cpp/work/.build/bin/vulkan-shaders-gen --glslc /usr/local/bin/glslc --input-dir /usr/ports/misc/llama-cpp/work/llama.cpp-b4285/ggml/src/ggml-vulkan/vulkan-shaders --output-dir /usr/ports/misc/llama-cpp/work/.build/ggml/src/ggml-vulkan/vulkan-shaders.spv --target-hpp /usr/ports/misc/llama-cpp/work/.build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.hpp --target-cpp /usr/ports/misc/llama-cpp/work/.build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp --no-clean
```

FreeBSD 14.1
clang-18
vulkan-headers-1.4.303
vulkan-loader-1.4.303

### Relevant log output

```shell
n/a
```

**Comment by jeffbolznv** [comment]
Can you try #10713?

**Comment by yurivict** [comment]
The build goes through with this PR.

**Comment by jeffbolznv** [comment]
Thanks, closing as fixed by #10713.
